GabbyTam_Fatima_functionpractice.js

```diff
@@ -0,0 +1,238 @@
+
+/* worked on part 1: 1-3 with breakout partner Daniel L */
+
+
+// source for string reversal loop
+https://www.freecodecamp.org/news/how-to-reverse-a-string-in-javascript-in-3-different-ways-75e4763c68cb/
+function reverseString(str) {
+    var newString = "";
+    for (var i = str.length - 1; i >= 0; i--) {
+        newString += str[i];
+        //console.log(newString);
+    }
+    if(newString== str){
+      return true;
+    } else {
+      return false;
+    }
+}
+
+// console.log(reverseString('mom'));
+reverseString('racecar');
```

Comment:
nice work! small thing to consider: what if your word's first letter were capitalized?